packages/contracts/test/voting/majority-voting.ts

```diff
@@ -103,5 +104,17 @@ describe('MajorityVotingMock', function () {
           votingSettings.minProposerVotingPower
         );
     });
+
+    it('should change the voting settings successfully', async () => {
+      expect(await votingBase.updateVotingSettings(votingSettings))
```

Comment:
should be  `await expect(`
```suggestion
      await expect(votingBase.updateVotingSettings(votingSettings))
```

Reply:
done in https://github.com/aragon/core/pull/177/commits/40752bfba0b8621a6b203498681a2ff2374adea9